src/org/omegat/Bundle.properties

```diff
@@ -2369,14 +2368,13 @@ TEAM_UPDATE_REPO_ERROR=Error updating repository. Skipping. Error: {0}
 # Save options
 SAVE_DIALOG_TITLE=Saving and Output Options
 PREFS_TITLE_SAVING_AND_OUTPUT=Saving and Output
-SAVE_DIALOG_INTERVAL_LABEL=Project data saving interval
-SAVE_DIALOG_DESCRIPTION=This option sets the project data saving interval, which also applies to team data synchronization.
+SAVE_DIALOG_INTERVAL_LABEL=Project save and team synchronization interval
```

Comment:
"save interval" is better than "saving interval", and I think data is redundant.

Reply:
Ok, but it’s not the whole project that’s saved. And with teams, it’s not the whole project and it’s more than the project memory that’s sync'ed...

Reply:
Does that matter from the user's point of view?

The end result is that everything that matters is saved/synched, and from an end user point of view, that essentially means the project is saved even if you're not technically saving every single file that's part of the project.

(If this was in the developer documentation, we'd have to be more specific, but either way "project data" is vague, and keeping it it would probably warrant restoring and editing the additional explanation you explicitly took out!)

Reply:
Now that I think about it, in terms of UI, what the user sees is that the translation is saved (and that's what they want to have saved), so what about "Translation save and synchronization interval"?